Core/include/Acts/EventData/SingleBoundTrackParameters.hpp

```diff
@@ -158,19 +176,18 @@ class SingleBoundTrackParameters {
   /// Access the time coordinate.
   Scalar time() const { return get<eBoundTime>(); }
 
-  /// Access the direction pseudo-rapidity.
-  Scalar eta() const { return -std::log(std::tan(get<eBoundTheta>() / 2)); }
```

Comment:
Do we not need this anywhere?

Reply:
No, this is not used anywhere in the code. All use cases are served by `VectorHelpers::eta(...)` already.

Reply:
Ok.